layouts/partials/css.html

```diff
@@ -17,7 +17,16 @@
 <!-- Custom CSS -->
 <link rel="stylesheet" type="text/css" href="{{ "css/styles.css" | relURL }}" />
 <link rel="stylesheet" type="text/css" href="{{ "css/shell.css" | relURL }}" />
+<<<<<<< HEAD
```

Comment:
This is an unresolved merge conflict. Normally git doesn't let you continue, somehow you managed to do that anyway @MarsBarLee. When this happens, you have to manually clean it up - deleting all `<<<<<`, `>>>>`, `=====` markers, and merging the content above and below `=====` so it makes sense again.

Reply:
Thanks for catching that! I cleaned up the merge conflicts in the other files but missed this one. I've resolved the merge conflict and committed it.